conditionals.js

```diff
@@ -199,15 +223,52 @@ function turnOffLight(lightName) {
 
 /* 
    -------YOUR CODE-----------------------------------------------------------
-*/
-function updateLights(somebodyIsHome, theyAreWatchingTV, itIsDarkOutside, theyAreCooking, theyWentToBed) {
-
-    // Write your code here!  You don't need to return anything, just call the given functions
+   // Write your code here!  You don't need to return anything, just call the given functions
     // You should be using if else statements and the function arguments
     // example of turning a light on
     turnOnLight("livingRoomLight");
     // example of turning off a light
     turnOffLight("livingRoomLight");
+        If it's dark out, the porch light should be on.  If it's not dark, then it's off.
+        If nobody home, then turn all the lights off (except the porch light). 
+        If someone is home but they haven't gone to bed, then the living room and dining room lights should be on.
+        if someone is cooking, then the kitchen light should be on.
+    However, if they are watching TV, then turn off the livingroom and dining room lights.
+*/
+
+
+function updateLights(somebodyIsHome, theyAreWatchingTV, itIsDarkOutside, theyAreCooking, theyWentToBed) {
+    if (itIsDarkOutside === true) {
```

Comment:
instead of doing `if (variable === true)` or `if (variable === false)` each time, you can simply do `if (variable)` and `if (!variable)` respectively